pnpm/pnpm-workspace.yaml

```diff
@@ -0,0 +1,2 @@
+packages:
+  - 'packages/**'
```

Comment:
Will we be able to specify dependency chains?

Reply:
In normal mode, `pnpm` figures out your real `package.json:dependencies` and links them. If you grep through our package names in the lock file you'll see `link` and such.

Reply:
Oh, if you mean for task execution chains (like rebuilding, etc.), pnpm isn't the tool to take care of that. This is just the baseline from which to copy in a new directory and add in wireit, nx, etc.